src/app.js

```diff
@@ -12,8 +12,7 @@ SenseHat = require('node-sense-hat');
 JoystickLib = SenseHat.Joystick;
 
 // And the handle to the LED matrix
-matrix = SenseHat.Leds;
-
+Matrix = SenseHat.Leds;
```

Comment:
I made this lower case to show the difference between a handle and a library, e.g. for `JoystickLib` you have to make another call to get the handle (`joystick`), but the matrix comes as a handle. I'm happy to change them all to lowercase if you think it's more clear, but I'd prefer to not have them all uppercase.

Reply:
feel free to change them all to whichever you prefer, I only changed this one because it required changing 1 line rather than 2 lines

Reply:
Gotcha - good catch. Let's change the other to lowercase?